src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -955,6 +955,13 @@ public void visitSources(SCMSourceObserver observer) throws IOException, Interru
                             if (!repo.getOwnerName().equals(repoOwner)) {
                                 continue; // ignore repos in other orgs when using GHMyself
                             }
+
+                            if (repo.isArchived()) {
+                                witness.record(repo.getName(), false);
+
+                                continue;
+                            }
+
```

Comment:
Can Org repo's also be archived?

Reply:
Yes :) We already did that quite often.

Reply:
@bjoernhaeuser 
Will need to put this if the "org" code path as well then?